component-lib/modal.html

```diff
@@ -0,0 +1,252 @@
+						</a>
+						<a class="drawer-item link-no-style" href="./typography.html">
+							<p>Typography</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./modal.html">
+							<p>Modal</p>
+						</a>
+					</div>
+				</div>
+			</div>
+
+			<div class="mobile-sidebar" id="mobile-sidebar">
+				<div class="drawer">
+					<div class="drawer-header">
+						<h4 class="typo-label">Getting Started</h4>
+					</div>
+					<div class="drawer-body">
+						<a class="drawer-item link-no-style" href="./installation.html">
+							<p>Installation</p>
+						</a>
+					</div>
+					<div class="drawer-header">
+						<h4 class="typo-label">Components</h4>
+					</div>
+					<div class="drawer-body">
+						<a class="drawer-item link-no-style" href="./avatar.html">
+							<p>Avatar</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./alert.html">
+							<p>Alert</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./badge.html">
+							<p>Badge</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./button.html">
+							<p>Button</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./cards.html">
+							<p>Card</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./images.html">
+							<p>Image</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./input.html">
+							<p>Input</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./typography.html">
+							<p>Typography</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./modal.html">
+							<p>Modal</p>
+						</a>
+					</div>
+				</div>
+			</div>
+			<div class="main-content">
+				<!-- componen-1 -->
+				<div class="component-container">
+					<div class="component-intro">
+						<div class="title">Modal</div>
+						<div class="sub-title">
+							The modal component provides a solid foundation for creating
+							dialogs, popovers, lightboxes, or whatever else.
+						</div>
+					</div>
+					<div class="component-example">
+						<div class="title">Example</div>
+						<div class="sub-title">
+							Click on live demo button on see example of model component.
+						</div>
+
+						<div class="component-display-container">
+							<button class="btn btn-primary" onclick="openModel()">
+								Live Demo
+							</button>
+							<div class="modal" id="modal">
+								<div class="modal-heading">
+									<p class="typo-title">Modal Title</p>
+									<button
+										class="btn btn-icon"
+										onclick="closeModel()"
+										class="closeModel"
+									>
+										<i class="fa fa-times fa-lg" aria-hidden="true"></i>
+									</button>
+								</div>
+								<div class="modal-body">
+									<p class="typo-subtext">
+										Model body goes here. The modal component provides a solid
+										foundation for creating dialogs, popovers, lightboxes, or
+										whatever else.
+									</p>
+								</div>
+								<div class="modal-footer">
```

Comment:
There could be more vertical spacing beetween footer and text

Reply:
Added more spacing in new commit.